### PR TITLE
allow psutil < 6

### PR DESCRIPTION
**Pull request by jfcg**
fixes autogluon build on MacOS 12.6 and Arm64

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of Apache-2.0 license.

### CODE REVIEW

File: timeseries/setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "numpy",
     "scipy",
     "pandas",
-    "psutil>=5.7.3,<5.9",
+    "psutil>=5.7.3,<6",
```
Comment (Innixma) [inline timeseries/setup.py:30]: `psutil>=5.7.3,<5.10` will be safer. We generally avoid capping at major versions as it is less stable than minor version capping.

**Review by Innixma — COMMENTED** [review]
Thanks for the contribution! Once my comment is addressed, happy to merge. Please also rebase the PR as we fixed a bug in the CI.

**Comment by Innixma** [comment]
I updated the PR to reflect my comment, will see if CI passes. I also fixed version range for networkx in timeseries which also had a similar problem.

**Comment by jfcg** [comment]
> I updated the PR to reflect my comment, will see if CI passes. I also fixed version range for networkx in timeseries which also had a similar problem.

Thank you.

File: timeseries/setup.py (hunk 1 above)
Comment (shchur) [reply timeseries/setup.py]: Actually, it seems that we don't use `psutil` anywhere in `timeseries`, so it's fine to remove this dependency completely from `timeseries/setup.py`.

**Review by shchur — APPROVED** [review]
LGTM!

**Comment by jfcg** [comment]
Hi @tonyhoo, this PR seems to have made it to `v0.6.2` but not mentioned in the release notes, fyi.

Thanks..

**Pull request merged**